### ISSUE TITLE
Remove featuring artists part from search query when searching for lyrics

### ISSUE THREAD

**Issue by ldokduy20** [Malopieds/InnerTune#403]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

When the app search for lyrics of a song, it often search with the song title as the query. But for different countries, YouTube have different titles for songs. We can solve this by searching for lyrics but omit the "feat..." part from the title (featured artists should be already included in the search query).

### Why do you want this feature?

It'll make searching for lyrics faster (I don't know anything about KuGou, but for LRCLIB, it finds lyrics faster).

### Additional information

For example:
United States: "Guess featuring Billie Eilish (feat. Billie Eilish)
Vietnam: "Guess featuring Billie Eilish (cùng với Billie Eilish)"

When searching lyrics with the title in Vietnam, it is slower and have less results.

**Comment by Malopieds** [comment]
What about three dots on lyrics (on the bottom right part of the lyrics), search and remove the unwanted part in the title?

**Comment by ldokduy20** [comment]
Yes, that’s the approach I used to make the lyrics appear. However, I find it a bit inconvenient, as there are instances where the lyrics won’t show up at all, even when I attempt to fetch them manually. If you believe this feature isn’t worth implementing, I’ll go ahead and close this. Thank you very much for the wonderful app and all your hard work!

**Comment by Malopieds** [comment]
> However, I find it a bit inconvenient, as there are instances where the lyrics won’t show up at all, even when I attempt to fetch them manually

Do you have any examples? If there are no lyrics to this song that could also explains it.

**Comment by ldokduy20** [comment]
https://github.com/user-attachments/assets/5593ef5b-01a5-4766-8ddd-e09b02b741f6

When I search for lyrics using the original title, LRCLIB ones don't show up. However, if I remove "cùng với" from the title, the lyrics appear. Unfortunately, the app now crashes whenever I try searching with the modified title again, so I couldn't get more examples. Sorry!
